### DISCUSSION TITLE
AITOOL not restarting deepstack on error 500's

### DISCUSSION THREAD

**Discussion by pocketgeek** [VorlonCD/bi-aidetection#228] [Q&A]
It looks like AITOOL isn't restarting deepstack on error 500's.  

![image](https://user-images.githubusercontent.com/10180058/113611852-4ff28580-960c-11eb-9b8e-46549c548e95.png)

Unless I don't have it configured correctly for this.
![image](https://user-images.githubusercontent.com/10180058/113611924-67317300-960c-11eb-9fbb-4b92bd487b32.png)

Hitting stop, then start fixes deepstack for awhile...until this happens again.  

--
Curtis Edge

**Comment by VorlonCD** [comment]
when it happens again, hit STDERR button to look at why it crashed
